tests/conftest.py

```diff
@@ -1,8 +1,23 @@
+import json
+import logging
 import os
 
+from labgrid.driver import ShellDriver
 import pytest
 
 
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(autouse=True, scope="module")
+def restart_qemu(strategy):
+    """Use fresh QEMU instance for each module."""
+    if strategy.status.name == "shell":
+        logger.info("Restarting QEMU before %s module tests.", strategy.target.name)
+        strategy.transition("off")
+        strategy.transition("shell")
```

Comment:
This seems to be unused currently?

Reply:
It is used, since it has `autouse=True, scope="module"` - so it's called before every test module. Similarly to `setup_module`, however `setup_module` doesn't have the required scope available.

Reply:
Ah I see :+1: